packages/clima_ui/android/app/src/main/kotlin/co/prestosole/clima/MainActivity.kt

```diff
@@ -0,0 +1,5 @@
+package co.prestosole.clima;
+import io.flutter.embedding.android.FlutterActivity
+
+class MainActivity: FlutterActivity() {
```

Comment:
This file isn't necessary at all (nor is the old Java one); you can change `.MainActivity` in `android/app/src/main/AndroidManifest.xml` to `io.flutter.embedding.android.FlutterActivity` and it should function identically.

Reply:
Should I open a separate PR for this?

Reply:
Certainly possible, but then you'll have to change this file back to Java, so might as well remove it altogether like I said, right?